R/pkg/R/DataFrame.R

```diff
@@ -1226,8 +1226,7 @@ setMethod("collect",
               # empty data.frame with 0 columns and 0 rows
               data.frame()
             } else if (useArrow) {
-              requireNamespace1 <- requireNamespace
-              if (requireNamespace1("arrow", quietly = TRUE)) {
+              if (requireNamespace("arrow", quietly = TRUE)) {
                 read_arrow <- get("read_arrow", envir = asNamespace("arrow"), inherits = FALSE)
```

Comment:
@MichaelChirico I believe this can be fixed as `arrow::read_arrow`. It was also a workaround. see also https://github.com/apache/spark/pull/25993

Reply:
Yep, I wasn't reading carefully enough.